.github/workflows/commitlint.yml

```
@@ -24,4 +25,4 @@ jobs:
         run: npm install --save-dev @commitlint/{config-conventional,cli}
 
       - name: Lint PR title
-        run: echo "${{ github.event.pull_request.title }}" | npx commitlint
+        run: echo "${{ github.event.pull_request.title }}" | npx commitlint@v17.7.2 
```

```suggestion
        run: echo "${{ github.event.pull_request.title }}" | npx commitlint
```
Right?  We shouldn't need this cuz it's spec'd in the package-llock.json file?